### PR TITLE
Add german translation

### PR DESCRIPTION
**Pull request by dlubitz**
Add german translation

### CODE REVIEW

File: Resources/Private/Translations/de/NodeTypes/LoginForm.xlf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='utf-8'?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+	<file original="" product-name="Flowpack.Neos.FrontendLogin" source-language="en" datatype="plaintext">
```
Comment (bwaidelich) [inline Resources/Private/Translations/de/NodeTypes/LoginForm.xlf]: Shouldn't that have a `target-language="de"` attribute as well?

File: Resources/Private/Translations/de/NodeTypes/LoginForm.xlf [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+			<target state="translated">Frontend Login-Formular</target></trans-unit>
+			<trans-unit id="groups.pluginSettings" xml:space="preserve" approved="yes">
+				<source>Login Settings</source>
+			<target state="translated">Login Einstellungen</target></trans-unit>
+			<trans-unit id="properties.redirectAfterLogin" xml:space="preserve" approved="yes">
+				<source>Page after login</source>
+			<target state="translated">Seite nach dem Login</target></trans-unit>
+			<trans-unit id="properties.redirectAfterLogout" xml:space="preserve" approved="yes">
+				<source>Page after logout</source>
+			<target state="translated">Seite nach dem Logout</target></trans-unit>
```
Comment (bwaidelich) [inline Resources/Private/Translations/de/NodeTypes/LoginForm.xlf]: <nitpick>Could you fix the indention of this? It's just cosmetical but makes it easier to read</nitpick>

File: Resources/Private/Translations/de/NodeTypes/LoginForm.xlf (hunk 1 above)
Comment (dlubitz) [inline Resources/Private/Translations/de/NodeTypes/LoginForm.xlf]: Don't know. I've created it with a tool (virtaal). Works in my projects as it is.

Comment (bwaidelich) [inline Resources/Private/Translations/de/NodeTypes/LoginForm.xlf]: OK, don't worry then

File: Resources/Private/Translations/de/NodeTypes/LoginForm.xlf [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
+        <source>Login Settings</source>
+      <target state="translated">Login Einstellungen</target></trans-unit>
+      <trans-unit id="properties.redirectAfterLogin" xml:space="preserve" approved="yes">
+        <source>Page after login</source>
+      <target state="translated">Seite nach dem Login</target></trans-unit>
+      <trans-unit id="properties.redirectAfterLogout" xml:space="preserve" approved="yes">
+        <source>Page after logout</source>
+      <target state="translated">Seite nach dem Logout</target></trans-unit>
+    </body>
+  </file>
```
Comment (bwaidelich) [inline Resources/Private/Translations/de/NodeTypes/LoginForm.xlf:18]: Not what I meant, but nevermind - especially as it's generated with a tool anyways ;)

**Comment by dlubitz** [comment]
Sorry, got it to late...

**Pull request merged**